sdk/storage/inc/datalake/path_client.hpp

```diff
@@ -32,56 +32,75 @@ namespace Azure { namespace Storage { namespace DataLake {
   {
     std::unique_ptr<Azure::Core::Http::BodyStream> Body;
     std::string AcceptRanges;
-    std::string CacheControl;
-    std::string ContentDisposition;
-    std::string ContentEncoding;
-    std::string ContentLanguage;
+    DataLakeHttpHeaders HttpHeaders;
     int64_t ContentLength = int64_t();
-    int64_t RangeOffset = int64_t();
-    int64_t RangeLength = int64_t();
-    std::string ContentType;
-    std::string ContentMD5;
+    Azure::Core::Nullable<int64_t> RangeOffset;
+    Azure::Core::Nullable<int64_t> RangeLength;
+    Azure::Core::Nullable<std::string> TransactionalMD5;
     std::string Date;
     std::string ETag;
     std::string LastModified;
     std::string RequestId;
     std::string Version;
     std::string ResourceType;
-    std::string LeaseDuration;
-    std::string LeaseState;
-    std::string LeaseStatus;
-    std::string ContentMd5;
+    Azure::Core::Nullable<std::string> LeaseDuration;
+    Azure::Core::Nullable<LeaseStateType> LeaseState;
+    Azure::Core::Nullable<LeaseStatusType> LeaseStatus;
```

Comment:
Should these two be nullable?

Reply:
Yes, if there is no lease, those won't be returned.

Reply:
I doubt that. can you give me an example of HTTP response?

Reply:
You are actually right, only the lease duration is optional, changed in next commit.